docs/docsite/rst/dev_guide/style_guide/spelling_word_choice.rst

```diff
@@ -57,7 +57,7 @@ Use "using" instead.
 
 Can/May
 ^^^^^^^
-Use "can" to describe actions or conditions that are possible. Use "may" only to describe situations where permission is being given. If either "can," "could," or "may" apply, use "can" because it is less tentative.
+Use "can" to describe actions or conditions that are possible. Use "may" only to describe situations where permission is being given. If either "can," "could," or "may" apply, use "can" because it's less tentative.
```

Comment:
Sorry, I wasn't clear. This `it's` should be replaced with `it is`, but the other one that specifically refers to the contraction `it's` in quotes should stay the same.

Reply:
I should have been more careful with the `spelling_word_choice.rst` file. Is it fine now?

Reply:
Yeah, looks alright.